testador/zzcineuci.sh

```diff
@@ -0,0 +1,67 @@
+UCI Cinema
+Nome_Filme   Duracao_Filme  Genero  Censura
+$ zzcineuci 17 | sed '1s/^UCI .*/UCI Cinema/;s/ *$//;s/Livre$/Censura/;s/[0-9][0-9] anos$/Censura/;s/Conteúdo Alternativo/Alternativo  /;s/[^ ]*  *Censura/Genero  Censura/;s/Duração(min) */Duracao_Filme  /;s/[0-9]\{1,\}  *Genero/Duracao_Filme  Genero/;s/.*Duracao/Nome_Filme   Duracao/' | uniq
+UCI Cinema
+Nome_Filme   Duracao_Filme  Genero  Censura
+$ zzcineuci 18 | sed '1s/^UCI .*/UCI Cinema/;s/ *$//;s/Livre$/Censura/;s/[0-9][0-9] anos$/Censura/;s/Conteúdo Alternativo/Alternativo  /;s/[^ ]*  *Censura/Genero  Censura/;s/Duração(min) */Duracao_Filme  /;s/[0-9]\{1,\}  *Genero/Duracao_Filme  Genero/;s/.*Duracao/Nome_Filme   Duracao/' | uniq
+UCI Cinema
+Nome_Filme   Duracao_Filme  Genero  Censura
+$ zzcineuci 20 | sed '1s/^UCI .*/UCI Cinema/;s/ *$//;s/Livre$/Censura/;s/[0-9][0-9] anos$/Censura/;s/Conteúdo Alternativo/Alternativo  /;s/[^ ]*  *Censura/Genero  Censura/;s/Duração(min) */Duracao_Filme  /;s/[0-9]\{1,\}  *Genero/Duracao_Filme  Genero/;s/.*Duracao/Nome_Filme   Duracao/' | uniq
+UCI Cinema
+Nome_Filme   Duracao_Filme  Genero  Censura
+$ zzcineuci 21 | sed '1s/^UCI .*/UCI Cinema/;s/ *$//;s/Livre$/Censura/;s/[0-9][0-9] anos$/Censura/;s/Conteúdo Alternativo/Alternativo  /;s/[^ ]*  *Censura/Genero  Censura/;s/Duração(min) */Duracao_Filme  /;s/[0-9]\{1,\}  *Genero/Duracao_Filme  Genero/;s/.*Duracao/Nome_Filme   Duracao/' | uniq
+UCI Cinema
+Nome_Filme   Duracao_Filme  Genero  Censura
+$ zzcineuci 22 | sed '1s/^UCI .*/UCI Cinema/;s/ *$//;s/Livre$/Censura/;s/[0-9][0-9] anos$/Censura/;s/Conteúdo Alternativo/Alternativo  /;s/[^ ]*  *Censura/Genero  Censura/;s/Duração(min) */Duracao_Filme  /;s/[0-9]\{1,\}  *Genero/Duracao_Filme  Genero/;s/.*Duracao/Nome_Filme   Duracao/' | uniq
+UCI Cinema
+Nome_Filme   Duracao_Filme  Genero  Censura
+$ zzcineuci 23 | sed '1s/^UCI .*/UCI Cinema/;s/ *$//;s/Livre$/Censura/;s/[0-9][0-9] anos$/Censura/;s/Conteúdo Alternativo/Alternativo  /;s/[^ ]*  *Censura/Genero  Censura/;s/Duração(min) */Duracao_Filme  /;s/[0-9]\{1,\}  *Genero/Duracao_Filme  Genero/;s/.*Duracao/Nome_Filme   Duracao/' | uniq
+UCI Cinema
+Nome_Filme   Duracao_Filme  Genero  Censura
+$ zzcineuci 24 | sed '1s/^UCI .*/UCI Cinema/;s/ *$//;s/Livre$/Censura/;s/[0-9][0-9] anos$/Censura/;s/Conteúdo Alternativo/Alternativo  /;s/[^ ]*  *Censura/Genero  Censura/;s/Duração(min) */Duracao_Filme  /;s/[0-9]\{1,\}  *Genero/Duracao_Filme  Genero/;s/.*Duracao/Nome_Filme   Duracao/' | uniq
+UCI Cinema
+Nome_Filme   Duracao_Filme  Genero  Censura
+$ zzcineuci 25 | sed '1s/^UCI .*/UCI Cinema/;s/ *$//;s/Livre$/Censura/;s/[0-9][0-9] anos$/Censura/;s/Conteúdo Alternativo/Alternativo  /;s/[^ ]*  *Censura/Genero  Censura/;s/Duração(min) */Duracao_Filme  /;s/[0-9]\{1,\}  *Genero/Duracao_Filme  Genero/;s/.*Duracao/Nome_Filme   Duracao/' | uniq
```

Comment:
Em vez de repetir o mesmo filtro sed em todas as linhas, podia guardar ele numa função, lá no início desse arquivo. Depois só faz `zzcineuci 25 | filtro`